streams/src/main/java/org/apache/kafka/streams/errors/internals/DefaultErrorHandlerContext.java

```diff
@@ -90,4 +105,8 @@ public String processorNodeId() {
     public TaskId taskId() {
         return this.taskId;
     }
+
+    public ProcessorContext processorContext() {
```

Comment:
Should this return `Optional<ProcessorContext>` given that it could be `null` ?

Reply:
done